docs/src/main/paradox/java/http/common/http-model.md

```diff
@@ -346,6 +346,14 @@ Similarily to the media types registration, you can register custom status codes
 
 @@snip [CustomStatusCodesExampleTest.java](../../../../../test/java/docs/http/javadsl/CustomStatusCodesExampleTest.java) { #application-custom-java }
 
+<a id="registeringcustommethod-java"></a>
```

Comment:
I actually don't know if we need to put these manually, do we @2m @johanandren ?